File: Assignment__26/Question1.java
/*
    Accept number from user and display below pattern.
    Input : 5
    Output : A B C D E
*/

import java.util.Scanner;

class Pattern
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        char ch = 'A';

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(ch + " ");
            ch++;
        }
    }
}

public class Question1
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the number: ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue);
    }
}

File: Assignment__41/Question1.c
#include<stdio.h>

void Display(int iNo)
{
    if(iNo == 0)  
    {
        return;
    }

    printf("%d", iNo);

    if(iNo > 1)
    {
        printf("\t*\t");
    }

    Display(iNo - 1);
}

int main()
{
    int iValue = 0;

    printf("Enter the number : ");
    scanf("%d", &iValue);

    Display(iValue);

    return 0;
}
